pkg/importer/dataStream.go

```diff
@@ -682,29 +682,84 @@ func (d *DataStream) isHTTPQcow2() bool {
 		len(d.Readers) == 2
 }
 
+func (d *DataStream) convertRawToQcow2Stream(dest string) error {
+	glog.V(3).Infoln("Validating qcow2 file")
+	err := qemuOperations.Validate(d.url.String(), "qcow2")
+	if err != nil {
+		return errors.Wrap(err, "Streaming image validation failed")
+	}
+	glog.V(3).Infoln("Doing streaming qcow2 to raw conversion")
+	err = qemuOperations.ConvertQcow2ToRawStream(d.url, dest)
+	if err != nil {
+		return errors.Wrap(err, "Streaming qcow2 to raw conversion failed")
+	}
+
+	return nil
+}
+
+func (d *DataStream) convertRawToQcow2(src, dest string) error {
```

Comment:
@igoihman  Should not these methods be in utils as generic methods and not relevant only to datastream?
Convert images
validate images
Resize images...

Reply:
they are wrappers for other functions and specific for datastream, they are not being used elsewhere.